src/Grphp/Client/Interceptors/Base.php

```diff
@@ -89,6 +89,24 @@ public function getFullyQualifiedMethodName(): string
         return $stub->getServiceName() . '/' . ucfirst($methodName);
     }
 
+    /**
+     * Gets the fully qualified service name, e.g. grphp.catalog.products
```

Comment:
Comment seems like it should be updated

Reply:
updated it and changed it to match the example from the actual tests